glusterd2/volgen2/clustergraph.go

```diff
@@ -72,6 +72,32 @@ func clusterGraph(volfile *Volfile, dht *Entry, vol *volume.Volinfo, nodeid uuid
 					decommissionedBricks = append(decommissionedBricks, name)
 				}
 			}
+
+			// FIXME: Change the name of thin arbiter option
+			thinarbiter, exists := vol.Options["cluster.replicate.thin-arbiter"]
+			if exists && thinarbiter != "" {
+				taParts := strings.Split(thinarbiter, ":")
```

Comment:
I would still like to keep it as 'host<:port>:/brick'  (whatever is the normal standard), so we can handle the case of different port of thin-arbiter from the day1 itself.
If its going to be difficult, it is ok, but assumption is most of the time user doesn't specify the port (in that case 24007 is fine), but if he specifies, we get control of handling this.